packages/client/src/runtime/core/types/Utils.ts

```diff
@@ -36,7 +36,7 @@ export type Narrow<A> = {
 // prettier-ignore
 export type Exact<A, W> = (A extends unknown ? (W extends A ? {
   [K in keyof W]: K extends keyof A ? Exact<A[K], W[K]> : never
-} : W) : never) | (A extends Narrowable ? A : never)
+} : W) : never) | (A extends Narrowable ? A : never) | ([W] extends [never] ? any : never)
```

Comment:
It's not really `Exact`'s role to do this, in fact I initially hoped that I could that `Prisma.Args` could simply return `any` if it reveived a bad input. However, because of the sensitivity of `Prisma.Args` and its interaction with `Prisma.Exact` as well as input inference, it's not possible to do so

Reply:
We can also see this differently: It does not make sense to pass `never` as the shape to be matched by `Exact`, so it can be seen as some kind of special edge case that would then allow `any` value to be matched.

Reply:
An alternative to this would be to let Accelerate handle the edge case (please read the slack conversation or explanation  in the issue), yet that also does not feel right if we can potentially make it slightly better. Arguably, Accelerate is also just using our API, so our role is to make it work as good as we possibly can with the described limitation.

Reply:
Let me know if you think there is a better way